### PR TITLE
Add post_send on ``device`` to ``AddVM``

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

**Comment by szok** [comment]
:ok:

File: src/ralph/ui/views/deploy.py [hunk 1]
```diff
@@ -503,6 +504,9 @@ def post(self, request, *args, **kwargs):
         resp['Location'] = LATEST_API.canonical_resource_for(
             'dev'
         ).get_resource_uri(device)
+        post_save.send(
```
Comment (mkurek) [inline src/ralph/ui/views/deploy.py:509]: add `device._handle_post_save = False` in line 488

Comment (ar4s) [inline src/ralph/ui/views/deploy.py:509]: I added `handle_post_save` to `create` method

**Comment by mkurek** [comment]
:ok:

**Pull request merged**